code/datums/keybinding/carbon.dm

```diff
@@ -89,6 +89,27 @@
 	user.mob?.a_intent_change(INTENT_HARM)
 	return TRUE
 
+/datum/keybinding/carbon/hold_throw_mode
+	key = "Space"
+	name = "hold_throw_mode"
+	full_name = "Hold throw mode"
+	description = "Hold this to turn on throw mode, and release it to turn off throw mode"
+	category = CATEGORY_CARBON
+	keybind_signal = COMSIG_KB_CARBON_HOLDTHROWMODE_DOWN
+
+/datum/keybinding/carbon/hold_throw_mode/down(client/user)
+	. = ..()
+	if(.)
+		return
+	var/mob/living/carbon/carbon_user = user.mob
+	carbon_user.throw_mode_on(THROW_MODE_HOLD)
+
+/datum/keybinding/carbon/hold_throw_mode/up(client/user)
+	. = ..()
+	if(.)
+		return
+	var/mob/living/carbon/carbon_user = user.mob
```

Comment:
> This can potentially cause runtimes for ghosts fiddling with the spacebar. I'd recommend adding if(iscarbon(user.mob)) or something similar!

Don't do an `if()` check. Check my latest commit.